code/go/0chain.net/smartcontract/dbs/event/blobber.go

```diff
@@ -21,6 +21,12 @@ type Blobber struct {
 	BlobberID string `json:"id" gorm:"uniqueIndex"`
 	BaseURL   string `json:"url" gorm:"uniqueIndex"`
 
+	//provider
+	LastHealthCheck int64 `json:"last_health_check"`
+	IsKilled        bool  `json:"is_killed"`
```

Comment:
```
can the same blobber be killed and shutdown at the same time? 
```
Unspecified, but as of this moment killed is a stronger version of shutdown and we could replace with an enumeration.
```
neither_shutdown_or_killed -> could be active or inactive
shutdown -> always inactive
killed -> always inactive
```
An issue here is that the designation that is used in the code is (active, inactive), which can change silently as global settings and provider settings change. So nothing quite fits. Which takes more space, two `bools` or an `int`?

Reply:
```
most likely we don't need gorm.Model there, since we don't use deletedAt and other fields on here
```
We will need to refactor the event database in several ways. Hopefully, load tests will help indicate areas where we have problems. But this should be for a separate PR.